CMakeLists.txt

```diff
@@ -126,6 +123,7 @@ ENDIF (UNIX)
 
 if (BUILD_SHARED_LIBS)
 	set (LIB_TYPE SHARED)
+	set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
Why only for shared fesapi?
According to https://stackoverflow.com/questions/3961446/why-does-gcc-not-implicitly-supply-the-fpic-flag-when-compiling-static-librarie "if you intend to link shared libraries against it, you need PIC code in your static library"
Since I don't know what fesapi users are going to do, I guess I don't want to take any risk and use fpic always.

Reply:
you are right I'll move it